.travis.yml

```diff
@@ -4,8 +4,6 @@ cache:
 install:
 - bundle install
 rvm:
-- 2.1
-- 2.2
```

Comment:
hmm we normally keep support for all versions of `!EOL?` versions per: https://github.com/sensu-plugins/documentation/blob/master/docs/FAQ.md#what-is-the-policy-on-supporting-end-of-lifeeol-ruby-versions

Per the ruby support dates: https://www.ruby-lang.org/en/downloads/branches/
```
Ruby 2.2
status: security maintenance
release date: 2014-12-25
EOL date: scheduled for 2018-03-31
```

In this case I am willing to make an exception because of the low volume this repo has had and it's close to an official EOL date. We will still version this as a major so that anyone who might be using this and pinning their gems will be unaffected until they are ready for it.

Reply:
<<~ operator in heredoc is supported by ruby 2.3

Reply:
<<~ operator in heredoc is supported by ruby 2.3